msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/ServiceFabricManagedIdentitySource.java

```diff
@@ -57,6 +57,7 @@ public ManagedIdentityResponse getManagedIdentityResponse(
             ManagedIdentityParameters parameters) {
 
         createManagedIdentityRequest(parameters.resource);
+        managedIdentityRequest.addTokenRevocationParametersToQuery(parameters);
```

Comment:
if this will go into each of the sources then the cae query params will be added only to those sources, right? in this case it will be added only to SF. Do we still need a CAE list? Ok to have a double check but wondering if it is really needed.

Reply:
In Java, the createManagedIdentityRequest is either created from ServiceFabric or AbstractManagedIdentity class. I am using this method in both the locations and as we add more sources it can go there. I like this way of having the list, I am also using it to add parameters for supported sources in the mocking.